packages/block-editor/src/components/block-draggable/draggable-chip.js

```diff
@@ -10,7 +10,8 @@ import { dragHandle } from '@wordpress/icons';
  */
 import BlockIcon from '../block-icon';
 
-export default function BlockDraggableChip( { count, icon } ) {
+export default function BlockDraggableChip( { count, icon, isPattern } ) {
+	const patternLabel = isPattern && __( 'Pattern' );
```

Comment:
Can we pass the `label` as a param here instead of `isPattern`. We will probably need it for the media tab too.